src/app/phase-bug-reporting/new-issue/new-issue.component.ts

```diff
@@ -53,6 +52,15 @@ export class NewIssueComponent implements OnInit {
         });
   }
 
+  canDeactivate() {
```

Comment:
Might wanna add simple unit tests for `canDeactivate()` and `isAttributeEditing()` !